c-lightning/2021-08-09-developer-call.md

```diff
@@ -0,0 +1,195 @@
+---
+title: c-lightning developer call (2021-08-09)
+transcript_by: Michael Folkson
+categories: ['meetup']
+tags: ['lightning', 'c-lightning']
+---
+
+Name: c-lightning developer call
+
+Topic: Various topics
+
+Location: Jitsi online
+
+Date: August 9th 2021
+
+Video: No video posted online
+
+The conversation has been anonymized by default to protect the identities of the participants. Those who have expressed a preference for their comments to be attributed are attributed. If you were a participant and would like your comments to be attributed please get in touch.
+
+# Individual updates
+
+Today I started to review some pull requests. I also finished my first draft of the paid order of the metrics plugin. I have a paid order make a chart. On [lnprototest](https://github.com/rustyrussell/lnprototest) I think we can remove the Postgres dependence but I haven’t tested it and I need to work around it.
```

Comment:
> I have a paid order make a chart.
to make a chart?

Reply:
I think this means the completion of a paid order makes a chart. Rather than being paid by an order to make a chart. So leaving this as is though I agree the wording could probably be improved.